### ISSUE TITLE
Does OVN-Kubernetes support DPDK interfaces?

### ISSUE THREAD

**Issue by TrevorTaoARM** [ovn-org/ovn-kubernetes#1055]
We know that OVS has DPDK interface support. Does the OVN-K8s support DPDK?
Can the "Shared" mode of OVN-k8s support the DPDK? 
Since I know the "Local" mode just uses Linux kernel Iptable NAT to do the routing.

**Comment by trozet** [comment]
No, this would require configuring netdev mode for the switches and setting up:
http://docs.openvswitch.org/en/latest/howto/userspace-tunneling/

There is some work underway to add regular netdev support for non-dpdk deployments. Perhaps you could piggy back off of that to add shared gateway dpdk support?

**Comment by trozet** [comment]
Also, ovn-kubernetes would not handle dpdk-binding the ports, I would expect something else to do that, and specify the dpdk interface to ovn-k8s. What do you think?
